frontend/webapp/src/pages/screen-recorder.js

```diff
@@ -0,0 +1,132 @@
+import { useState, useRef } from "react";
+
+export default function ScreenRecorder() {
+  const [isRecording, setIsRecording] = useState(false);
+  const [videoUrl, setVideoUrl] = useState(null);
+  const mediaRecorder = useRef(null);
+  const recordedChunks = useRef([]);
+
+  const startRecording = async () => {
+    try {
+      // Request screen recording
+      const screenStream = await navigator.mediaDevices.getDisplayMedia({
```

Comment:
@Swapnilden Did you fix below issue ?
 "even after the recording stops, the browser url window still has a 'recording' icon and shows "this tab is sharing your screen' "

Reply:
@janakiramg yes